src/views/CreditTracker.vue

```diff
@@ -23,6 +25,10 @@
 				</b-input-group>
 				<b-button variant="success" @click="addCourse">Add</b-button>
 			</div>
+			<div>
+			
+				Please check on  : <a href="https://ugradcalendar.uwaterloo.ca/" class="linkified" target="_blank">https://ugradcalendar.uwaterloo.ca/</a> for more course catogary infomation.
+				</div>
```

Comment:
```
<div>
<b-form-select>...</b-form-select>
<label style="
    font-size: 12px;
    float: left;
">Please visit {{url}} for more info
</label>
</div>

```
You can change line [#22](https://github.com/andythsu/WaterlooStudies/pull/13/files#diff-b5be0abcfcab0ef1ff76b1776ce91a625d13552c3fff739a12cb136d3e86185fR24) to this to make it display better